Nuake/src/Core/OS.cpp

```diff
@@ -0,0 +1,17 @@
+﻿#include "OS.h"
+
+#include <chrono>
+#include "String.h"
```

Comment:
I would group header files like this:

#include "OS.h"
#include "String.h"

then std includes